src/main/java/net/geforcemods/securitycraft/entity/EntitySentry.java

```diff
@@ -207,6 +210,24 @@ public void toggleMode(EntityPlayer player)
 		else
 			SecurityCraft.network.sendToAll(new PacketCInitSentryAnimation(getPosition(), true, mode == 0));
 	}
+	
+	/**
+	 * Sets this sentry's mode to the next one and sends the player a message about the switch

```

Comment:
"Sets this sentry's mode to the given mode (or 0 if the mode is not one of 0, 1, 2) and sends the player a message about the switch"